mgmt2/rpc/jsonrpc/JsonRPC.h

```diff
@@ -33,17 +33,17 @@ extern RPCRegistryInfo core_ats_rpc_service_provider_handle;
 /// @see JsonRPCManager::add_method_handler for details
 template <typename Func>
 inline bool
-add_method_handler(const std::string &name, Func &&call, const RPCRegistryInfo *info = nullptr)
+add_method_handler(const std::string &name, Func &&call, const RPCRegistryInfo *info, TSRPCHandlerOptions const &opt)
```

Comment:
Since this seems to get called with a string literal, you could avoid a heap allocation if the type was std::string_view.

Reply:
Thanks Walt.
yes, I would probably have them constant defined and not use the name as literal in the call anyway, I wasn't very worry about this as it only happens at startup and at the end it will allocate to hold it internally. 
I may add a new pr with some tweaks I have noted, takling this too possibly.